client-app/core/utilities/properties/index.ts

```diff
@@ -18,7 +18,7 @@ export function getPropertyValue(property: Property): string {
       return n(property.value);
 
     default:
-      return String(property.value);
+      return String(property.value ?? "-");
```

Comment:
Move to compare please as it's compare specific and we don't want to work it that way in other pages such as product page or product card on category page